examples/java/build.gradle

```
@@ -26,6 +26,10 @@ allprojects {
       endWithNewline()
     }
   }
+  tasks.withType(AbstractArchiveTask).configureEach {
+    preserveFileTimestamps = false
+    reproducibleFileOrder = true
+  }
```

Newline please :pray: 
```suggestion

    tasks.withType(AbstractArchiveTask).configureEach {
    preserveFileTimestamps = false
    reproducibleFileOrder = true
  }
```